### PR TITLE
Update consensus key change

### PR DESCRIPTION
**Pull request by brentstone**
## Describe your changes
Add an e2e test and require to sign the tx with the new consensus key.

## Indicate on which release or other PRs this topic is based on
v0.28.1

## Checklist before merging to `draft`
- [x] I have added a changelog
- [x] Git history is in acceptable state

### CODE REVIEW

**Review by tzemanovic — APPROVED** [review]

File: apps/src/lib/client/tx.rs [hunk 1]
```diff
@@ -459,17 +459,14 @@ pub async fn submit_change_consensus_key(
...
             display_line!(
                 namada.io(),
-                "  Consensus key \"{}\"",
-                consensus_key_alias
+                "New consensus key stored with alias \
+                 \"{consensus_key_alias}\". It will become active \
+                 {EPOCH_SWITCH_BLOCKS_DELAY} blocks before pipeline offset \
+                 from the current epoch, at which point you'll need to give \
+                 the new key to CometBFT in order to be able to sign with it \
+                 in consensus.",
```
Comment (brentstone) [inline apps/src/lib/client/tx.rs:469]: can we somehow make this message dependent on if the transaction is valid or not? Playing around and testing to make sure if validator-1 submits the consensus key change for validator-0, then the tx is invalid. However, we still currently print this message when it is invalid.

Comment (tzemanovic) [reply apps/src/lib/client/tx.rs:469]: ah good catch! Looks like we need to do this for couple other txs too, so I'll do it in a follow up PR - the client output could be overall improved

Comment (tzemanovic) [reply apps/src/lib/client/tx.rs:469]: also noticed that for some reason, the storage keys for this inner tx show up in the client result as empty (they don't show in `Transaction applied with result:` info when they're empty) - I can see them populated in the node, so I'll open another issue to find why

Comment (brentstone) [reply apps/src/lib/client/tx.rs:469]: Opened #2267 for this

**Review by tzemanovic — APPROVED** [review]

**Comment by tzemanovic** [comment]
fixed the issue in benches [1c4f085](https://github.com/anoma/namada/pull/2218/commits/1c4f085590c283fe9679662067ae6797a3ebfdf2) - the tx needed the extra sig with the new consensus key

**Pull request merged**